spec/react_on_rails/support/generator_spec_helper.rb

```diff
@@ -8,7 +8,7 @@
 # Expects an array of strings, such as "--redux"
 def run_generator_test_with_args(args, options = {})
   prepare_destination # this completely wipes the `destination` directory
-  simulate_existing_file(".gitignore")
+  simulate_existing_file(".gitignore") unless options.key?(:has_errors)
```

Comment:
I would use `fetch(:has_errors, false)` here, as this would be the wrong behavior if I passed `has_errors: false`.

Also, I wouldn't call it `has_errors`, I would just do it like: `gitignore: false` as this is easier to reason about